content/en/docs/reference/config-api/apiserver-eventratelimit.v1alpha1.md

```diff
@@ -105,17 +105,17 @@ allowance of burst queries.</p>
 </tbody>
 </table>
 
-## `LimitType`     {#evenratelimit-admission-k8s-io-v1alpha1-LimitType}
+## `LimitType`     {#eventratelimit-admission-k8s-io-v1alpha1-LimitType}
     
 (Alias of `string`)
 
 **Appears in:**
 
-- [Limit](#evenratelimit-admission-k8s-io-v1alpha1-Limit)
+- [Limit](#eventratelimit-admission-k8s-io-v1alpha1-Limit)
 
 
 <p>LimitType is the type of the limit (e.g., per-namespace)</p>
 
 
 
-  
+  
```

Comment:
Many files in `content/en/docs/reference/config-api` have this 3 newline + 2 space.

Not sure what should we do for this line.
1) Remove two spaces (current changes)
2) Leave it as is
3) Remove the entire line (newline + 2 space)

Reply:
This empty lines are generated from go-template.